library/src/cache/patch_manager.rs

```diff
@@ -445,7 +478,8 @@ impl ManagePatches for PatchManager {
             );
         }
 
-        self.patches_state.last_attempted_patch = Some(next_boot_patch);
+        self.patches_state.last_attempted_patch = Some(next_boot_patch.clone());
+        self.patches_state.currently_booting_patch = Some(next_boot_patch.clone());
```

Comment:
I'm confused.  Didn't we already have the concept of a currently booting patch?  How did we successfully mark them failing on failed boot before?

Reply:
Please leave yourself a note that you intend to delete the last_attempted_patch.

Reply:
Discussed on VC, this is going away in https://github.com/shorebirdtech/updater/pull/186